.github/workflows/image-check.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-go@v2
         with:
-          go-version: '1.19.9'
+          go-version: '1.20.0'
```

Comment:
since all other WFs are using `'1.20.x'` i feel like we should update to be consistent here.